_layouts/series.html

```diff
@@ -71,19 +71,19 @@ <h1 data-media-title="{{page.title}}" data-media-id="{{page.id}}" class="font-fa
         {% assign tabs_array = "" | split: "|" %}
   
         {% if page.videos.size > 0 %}
-          {% assign tabs_array = tabs_array | push: "Online" %}
+          {% assign tabs_array = tabs_array | push: "Live Service" %}
         {% endif %}
   
         {% if page.live_messages.size > 0 %}
-          {% assign tabs_array = tabs_array | push: "At Sites" %}
+          {% assign tabs_array = tabs_array | push: "Message Only" %}
         {% endif %}
   
         <div>
           <crds-tabs tabs='{{ tabs_array | jsonify }}' navigation-class="align-tabs-left component-header" horizontal-only>
-            <div slot="tab-online">
+            <div slot="tab-full-service">
```

Comment:
i think this needs to use the same namespacing as below this should be live-service